mne/decoding/receptive_field.py

```diff
@@ -79,16 +82,14 @@ class ReceptiveField(BaseEstimator):
            7, 13654 (2016). doi:10.1038/ncomms13654
     """
 
-    def __init__(self, tmin, tmax, sfreq, feature_names=None,
-                 estimator=None, scoring='r2'):  # noqa: D102
+    def __init__(self, tmin, tmax, sfreq, feature_names=None, estimator=None,
+                 fit_intercept=True, scoring='r2'):  # noqa: D102
         self.feature_names = feature_names
         self.sfreq = float(sfreq)
         self.tmin = tmin
         self.tmax = tmax
         self.estimator = 0. if estimator is None else estimator
-        if scoring not in _SCORERS.keys():
-            raise ValueError('scoring must be one of %s, got'
-                             '%s ' % (_SCORERS.keys(), scoring))
```

Comment:
I was told no type checking or anything should be done in `__init__` so I moved this to `fit`